lib/types.ts

```diff
@@ -8,12 +8,15 @@ declare module 'tmp' {
     export const tmpdir: string;
 }
 
-export {Suite as HermioneSuite} from 'hermione';
+export {Suite as TestplaneSuite} from 'hermione';
 
-export interface HermioneTestResult extends HermioneTestResultOriginal {
+export interface HermioneTestResult extends TestResult {
     timestamp?: number;
 }
 
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+export interface TestplaneTestResult extends HermioneTestResult {}
```

Comment:
These two are separated, so we could add something to `TestplaneTestResult` while it still does not exist on `HermioneTestResult`